corehq/apps/es/case_search.py

```diff
@@ -254,20 +266,24 @@ def reverse_index_case_query(case_ids, identifier=None):
     )
 
 
+def case_property_not_set(case_property_name):
```

Comment:
Why is this extracted?  Just for readability?  If so, can you prefix it with an underscore?  We generally try to treat "not set" and "set to blank" the same, and I'm worried having this filter available will encourage people to use the wrong one by mistake.